src/index-helper.js

```diff
@@ -18,3 +18,7 @@ export default function muiSetup(isActive, sproutBase) {
 
   return { theme, generateClassName };
 }
+
+export function shouldUpdateData(layout, tableData) {
+  return !tableData || JSON.stringify(layout.qHyperCube.qSize) !== JSON.stringify(tableData.size);
+}
```

Comment:
This works, but probably needs to be extended when there are other things that would need to reload the data

Reply:
yea, can't know for sure that the size changes just because the data changes.

Reply:
True... A different approach would be actually looking at what `manageData` returns and compare it to the old data. This would be slower, but covers what we need I think

Reply:
In general you should update if the layout hook fires, not entirely sure what difference this makes.

Reply:
The layout keeps track of what the active selections are, so layout is updated every time a cell is clicked, thus renders the whole tree